src/core/p5.Graphics.js

```diff
@@ -65,7 +65,53 @@ p5.Graphics = function(w, h, renderer, pInst) {
 p5.Graphics.prototype = Object.create(p5.Element.prototype);
 
 /**
+ * Removes a Graphics object from the page and frees any resources
+ * associated with it.
+ *
  * @method remove
+ *
+ * @example
+ * <div><code>
+ * var bg;
+ * function setup() {
+ *   createCanvas(100, 100);
+ *   stroke(255);
+ *   colorMode(HSB);
+ *
+ *   // create and draw the background image
+ *   bg = createGraphics(100, 100);
+ *   bg.background(100);
+ *   bg.colorMode(HSB);
+ *   bg.strokeWeight(3);
+ *   for (var i = 0; i < 100; i++) {
+ *     bg.stroke(random(0, 255), 255, 255);
+ *     bg.line(random(0, 100), random(0, 100), random(0, 100), random(0, 100));
+ *   }
+ * }
+ * function draw() {
+ *   var t = millis() / 1000;
```

Comment:
these 5 lines are a bit complicated, is there a way to simplify to make the example more clear?